form_tester/build.gradle

```diff
@@ -61,4 +61,5 @@ dependencies {
     implementation 'com.android.support:support-v4:28.0.0'
     implementation 'com.android.support:recyclerview-v7:28.0.0'
     implementation 'com.google.code.gson:gson:2.8.6'
+    implementation 'org.smartregister:opensrp-client-utils:0.0.4-SNAPSHOT'
```

Comment:
Is this in use?

Reply:
This module contains the Form class used in forms.

Reply:
Do any of the changes here require this?